### PR TITLE
WIP :: Do not merge :: Testing Fortran support

### LINKED ISSUE #11: Include Fortran bindings
A number of scientific codes use the Fortran bindings for HDF5. Would it be possible to update this recipe to also make sure the Fortran bindings are installed?

### PR DESCRIPTION
**Pull request by ocefpaf**
Just testing what would take to add Fortran support to `hdf5`.

Closes #11

### CODE REVIEW

**Comment by jakirkham** [comment]
Looks basically like there are problems with `.la` ( `libtool` ) files shipped with `gcc`.

**Comment by astrofrog** [comment]
@ocefpaf - thanks for working on this! I think you need to add ``libgfortran`` to the dependencies?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,14 +32,14 @@ requirements:
...
 
 test:
   requires:
-    - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
-    - python 3.5.*  # [win and py35]
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
 
   commands:
     # Verify UNIX CLI tools.
```
Comment (astrofrog) [inline recipe/meta.yaml]: @ocefpaf - once you update this PR, you could add ``h5fc`` as a command to test here too

**Comment by ocefpaf** [comment]
> @ocefpaf - thanks for working on this! I think you need to add libgfortran to the dependencies?

`libgcc` is also `libgfortran` and at the time it was our only option on `OS X`. But I see you made progress in #28 so I am closing this.

Thanks @astrofrog!

**Pull request closed without merging**